src/psyclone/tests/parse_test.py

```diff
@@ -1,10 +1,40 @@
 # -----------------------------------------------------------------------------
-# (c) The copyright relating to this work is owned jointly by the Crown,
-# Met Office and NERC 2015.
+# BSD 3-Clause License
+#
+# Copyright (c) 2017, Science and Technology Facilities Council
 # However, it has been created with the help of the GungHo Consortium,
```

Comment:
Please remove these two lines (5 and 6).

Reply:
Removed.